src/framework/uicomponents/qml/MuseScore/UiComponents/IncrementalPropertyControl.qml

```diff
@@ -49,7 +49,7 @@ Item {
     implicitHeight: 30
     implicitWidth: parent.width
 
-    navigation.name: root.objectName != "" ? root.objectName : "IncrementalControl"
+    navigation.name: root.objectName ?? "IncrementalControl"
```

Comment:
My opinion about '??'
https://github.com/musescore/MuseScore/pull/11314#pullrequestreview-957450120

I suggest not to use '??'

Reply:
Okay, I reverted it to the `Boolean(...) ? ... : ...` solution. I think we shouldn't let a buggy IDE limit which language features we use though...